### PR TITLE
add global test

### PR DESCRIPTION
**Pull request by alexg9010**
This PR is supposed to address #97 . 
`make distcheck` passes, however, there are a few things that still need to be addressed in the future (see https://github.com/BIMSBbioinfo/pigx_sarscov2_ww/issues/102)

### CODE REVIEW

**Comment by alexg9010** [comment]
Hi @rekado and @vfschumann ,

are you okay with merging this branch in its current state?
I think to continue with newer feature branches it would be great if this becomes part of `main`.

File: Makefile.am [hunk 1]
```diff
@@ -8,11 +8,18 @@ dist_pkglibexec_SCRIPTS = \
...
+	scripts/get_qc_table.py \
+	scripts/overview_QC_table.R \
+	scripts/parse_vep.py \
+	scripts/pooling.R \
+	scripts/pred_mutation_increase.R \
   scripts/renderReport.R \
-  scripts/get_qc_table.py \
-  scripts/parse_vep.py \
+	scripts/sample_coverage_score.R \
+	scripts/table_extraction.R \
```
Comment (rekado) [inline Makefile.am]: What's up with the indentation change here?

File: Makefile.am [hunk 2]
```diff
@@ -34,23 +41,35 @@ dist_doc_DATA = 									\
...
+	tests/sample_data/reads/Test0_R1.fastq.gz \
+	tests/sample_data/reads/Test0_R2.fastq.gz \
+	tests/sample_data/reads/Test2_R1.fastq.gz \
+	tests/sample_data/reads/Test2_R2.fastq.gz \
+	tests/sample_data/reads/Test5_R1.fastq.gz \
+	tests/sample_data/reads/Test5_R2.fastq.gz \
+	tests/sample_data/NC_045512.2.fasta \
+	tests/sample_data/nCoV-2019_NCref.bed \
+	tests/sample_data/Covid_CG_NTmutation_t07.bed \
+  tests/sample_data/mutation_sheet_211006_covidCG_NT_location.csv
```
Comment (rekado) [inline Makefile.am:60]: ...and here?  It's a bit inconsistent, too.

File: Makefile.am [hunk 3]
```diff
@@ -34,23 +41,35 @@ dist_doc_DATA = 									\
...
+
 
 AM_TESTS_ENVIRONMENT = srcdir="$(abs_top_srcdir)" builddir="$(abs_top_builddir)" PIGX_UNINSTALLED=1 PIGX_UGLY=1
 
 TESTS = \
-  tests/tests/vep.sh 
+  tests/tests/vep.sh \
+	test.sh
```
Comment (rekado) [inline Makefile.am]: and here too.

File: Makefile.am [hunk 4]
```diff
@@ -34,23 +41,35 @@ dist_doc_DATA = 									\
...
 
 clean-local:
 	-rm -rf .snakemake
 
-CLEANFILES = $(nodist_bin_SCRIPTS) config.json		
+CLEANFILES = $(nodist_bin_SCRIPTS) config.json
```
Comment (rekado) [inline Makefile.am:71]: Yay :)

**Review by rekado — COMMENTED** [review]
This looks good to me.  I'm just picky about spurious indentation changes.  Other than that it's fine to merge.

**Comment by alexg9010** [comment]
Great! I will take care of the indentations.

Sent from mobile.

rekado ***@***.***> schrieb am Mo., 7. März 2022, 16:18:

**Pull request merged**